gather_similarites.py

```diff
@@ -0,0 +1,68 @@
+from rdkit import Chem
+import numpy as np
+import tensorflow as tf
+
+import parse_sdf_utils
+import train_test_split_utils
+import feature_utils
+import mass_spec_constants as ms_constants
+import molecule_predictors
+import similarity as similarity_lib
+
+
+def make_spectra_array(inchikey, inchikey_dict):
```

Comment:
It would be a more straightforward API if make_spectra_array took a list and in the upstream code you called it using
make_spectra_array(inchikey_dict[inchikey]).

Reply:
done.